src/dagre-wrapper/markers.js

```diff
@@ -133,6 +133,7 @@ const point = (elem, type) => {
     .attr('markerWidth', 12)
     .attr('markerHeight', 12)
     .attr('orient', 'auto')
+    .attr('style', 'fill:#f00;stroke:#f00')
```

Comment:
That's a way we can set a color on the arrow head. Of course we don't want to hardcode that here. I think what could work is creating markers here for all types of arrow heads (as in having different styles) with different ids (append an index?) and then use the appropriate arrow head within the graph.

Reply:
It is okay to open a pull request to discuss the solution. It is a good practice to mark it as draft PR. 

Note that getting these styles from the theming engine is much preferred. Check src/style.js and src/diagrams/flowchart/styles.js/getStyles (line 36)

Hope this helps you out

Reply:
Thanks for the hints